Acrolinx.Sidebar/AcrolinxSidebar.cs

```diff
@@ -395,6 +395,12 @@ public String Check(IDocument document)
             string checkId = check.checkId;
             return checkId;
         }
+        public void CancelCheck()
+        {
+            var code = "new function(){ return acrolinxSidebar.onGlobalCheckRejected(); }";
```

Comment:
I think the function will not be executed... you just evaluate it.
I think what you wanted to do is:
`acrolinxSidebar.onGlobalCheckRejected();`
without all that function stuff...

Reply:
Done